provider/cldr/src/transform/plurals.rs

```diff
@@ -185,6 +186,18 @@ pub(self) mod cldr_json {
     }
 }
 
+pub(crate) fn parse_plural_category(category: &str) -> Option<PluralCategory> {
+    match category {
+        "zero" => Some(PluralCategory::Zero),
+        "one" => Some(PluralCategory::One),
+        "two" => Some(PluralCategory::Two),
+        "few" => Some(PluralCategory::Few),
+        "many" => Some(PluralCategory::Many),
+        "other" => Some(PluralCategory::Other),
+        _ => None,
+    }
+}
```

Comment:
`chore`: Please, add it to `PluralCategory` impl.

Reply:
Done